pkg/engine/rule/mock.go

```diff
@@ -2,8 +2,30 @@ package rule
 
 import (
 	"github.com/kubescape/kapprofiler/pkg/collector"
+	corev1 "k8s.io/api/core/v1"
 )
 
+type EngineAccessMock struct {
+}
+
+func (e *EngineAccessMock) GetPodSpec(podName, namespace, containerID string) (*corev1.PodSpec, error) {
+	podSpec := corev1.PodSpec{}
+	podSpec.Containers = []corev1.Container{
+		{
+			Name:  "test",
+			Image: "test",
+			VolumeMounts: []corev1.VolumeMount{
```

Comment:
Did you take care of SubPath and the rest of the variations?

Reply:
SubPath will be taking care of automatically because it checks against the path inside of the container.